keycloak.tf

```diff
@@ -92,8 +92,8 @@ resource "aws_ecs_task_definition" "keycloak-ecs-taskdef" {
           "hostPort": 9000
         }
       ],
-      "cpu": 512,
-      "memory": 2048,
+      "cpu": 1024,
```

Comment:
suggestion: should these be local variables so that we don't miss updating them in the future?

Reply:
Good call! https://github.com/mbta/terraform-keycloak-sso/pull/31/commits/cbd17e2247444ecd66fe8da687eba1978b3c983e